WTWSMS/common/traits/02_traits.txt

```diff
@@ -1093,10 +1095,15 @@ schemer = {
 theologian = {
 	lifestyle = yes
 	
-	learning = 2
-	intrigue = 1
+	learning = 4
+	diplomacy = 2
```

Comment:
It is a life-style trait though. They have certain boosts, to not favour one part more than the other. We don't want the only viable trait to be Theologian. As for intrigue, I'm unsure on why it is added, but I think it may be because Theologians are prone to be prominent at court and did take part in intrigue.